docs/TRANSLATING.md

```diff
@@ -2,13 +2,29 @@
 
 This guide is written to provide instructions from scratch for contributing to the translation of the Jamulus application to other languages.
 
-The code for Jamulus is open source, and is managed and made available via the Github site.
+The code for Jamulus is open source, and is managed and made available via the Github site. However, translators are not required to use GitHub. Jamulus is available on [Hosted Weblate](https://hosted.weblate.org/projects/jamulus/).
 
-For completeness, this document describes the use both of GitHub (using `git`) and of _Qt Linguist_.
+You have two options to translate Jamulus:
+1. Using [Weblate](https://hosted.weblate.org/projects/jamulus/jamulus-app/) (online translation, easier and more feature rich. However, as of 09/2022, it will show your e-mail address on GitHub.) (recommended)
+2. Using git directly (more advanced, but you can test your changes on your machine locally)
 
 ---
 
-## Introduction
+## Introduction (using Weblate)
+
+Translators should - if possible - have a GitHub account to communicate, as described in the **1. Setting up** paragraph in the git section below. This account should be used to [create an account on Hosted Weblate](https://hosted.weblate.org/accounts/register/). Ensure that the Jamulus main developers are aware of your intentions to translate to your chosen language beforehand (e.g. by [creating or commenting on a respective issue during the release process for your language on GitHub](https://github.com/jamulussoftware/jamulus/issues)). This ensures you will have proper access and support provided. Have a look at the [adding a new language (checklist)](#adding-a-new-language-checklist) for what to expect.
```

Comment:
```suggestion
Translators should  (if possible) have a GitHub account to communicate, as described in the **1. Setting up** paragraph in the Git section below. This account can also be used to [create an account on Hosted Weblate](https://hosted.weblate.org/accounts/register/). Ensure that the Jamulus main developers are aware of your intentions to translate to your chosen language beforehand (e.g. by [creating or commenting on a respective issue during the release process for your language on GitHub](https://github.com/jamulussoftware/jamulus/issues)). This ensures you have proper access and that support is provided. Have a look at the [adding a new language (checklist)](#adding-a-new-language-checklist) for what to expect.
```

Reply:
Applied in a slightly different way (can --> should) as stronger suggestion.

Reply:
That also breaks the FSF and DFSG guidelines, because it steers the user towards use of closed source software.
I don't understand how using GitHub OAuth is somehow better.